internal/clients/team5/forage.go

```diff
@@ -342,22 +347,15 @@ func (c *client) MakeForageInfo() shared.ForageShareInfo {
 	if c.getTurn() > c.config.InitialForageTurns { // for the turns we are NOT doing initial forage
 		for _, FOutcome := range c.forageHistory {
 			for _, returns := range FOutcome {
-				for _, team := range c.getAliveTeams(false) { // For all alive teams
-					if c.getTurn() > c.config.DeerTurnsToLookBack { // Turns greater than look back
-						if returns.turn > c.getTurn()-c.config.DeerTurnsToLookBack &&
-							returns.turn <= c.getTurn() &&
-							returns.team == team { // If a certain team within a certain range
+				if c.getTurn() > c.config.DeerTurnsToLookBack && // prevent looking at negative turns
+					c.getTurn() >= c.getTurn()-c.config.DeerTurnsToLookBack { // Turns greater than look back
```

Comment:
shouldn't the RHS be FOutcome.turn

Reply:
fixed, thanks for the spot. Could you also merge for me? I have no write acccess